src/app/settings/shelf-settings/components/static-buttons-configuration-card/static-buttons-configuration-card.component.html

```diff
@@ -0,0 +1,21 @@
+<sofie-card class="pt-2 pl-0">
+    <ng-container cardHeader>
+        <span>
+            <span class="card-header" i18n>static-buttons-configuration.card-header</span>
+        </span>
+        <sofie-icon-button [icon]="Icon.PEN" [iconSize]="IconSize.M" [tooltip]="'Edit Static Buttons'" (click)="openEditStaticButtons()"></sofie-icon-button>
+    </ng-container>
+        <div class="static-buttons-display">
+            <ng-container *ngIf="selectedStaticActions.length !== 0 else noStaticButtons">
+                <div class="button-grid">
+                    <ng-container *ngFor="let action of selectedStaticActions">
+                        <sofie-action-box [action]="action"></sofie-action-box>
+                    </ng-container>
+                </div>
+            </ng-container>
```

Comment:
You don't need these ng-containers, you can put the directives directly on the markup